### DISCUSSION TITLE
Keyball39の特定のキーが反応しない（ロープロファイル）

### DISCUSSION THREAD

**Discussion by kodomon** [Yowkees/keyball#477] [Q&A]
お世話になっております。
keyball39を購入させていただき、組立てましたが、
下記写真で赤丸で示した部分（ロープロファイルのキーの部分）が反応しません。
表側からピンセットでショートさせても反応がありません。
[[3-7．ProMicroの書き込みと動作確認]](https://github.com/Yowkees/keyball/blob/main/keyball61/doc/rev1/buildguide_jp.md#3-7promicro%E3%81%AE%E6%9B%B8%E3%81%8D%E8%BE%BC%E3%81%BF%E3%81%A8%E5%8B%95%E4%BD%9C%E7%A2%BA%E8%AA%8D)でのピンセットによる導通確認では反応があったと記憶しております。
また、試しにCherry MX互換キースイッチを取り付けてキーを押したところ反応しました。
対処法がわかりましたらお教えいただけますと幸いです。
お手数ですが、ご確認よろしくお願いいたします。
![【加工後】keyball39_left](https://github.com/Yowkees/keyball/assets/157866167/a46fd353-255d-43b2-a65a-bf8ed1fbb11a)
![【加工後】keyball39_right](https://github.com/Yowkees/keyball/assets/157866167/451074a6-d427-4a37-9469-2e6e6bd1a0a6)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

MXスイッチでは動作するということで、chocキーソケットに問題がありそうです。
写真では大丈夫そうに見えますが、念のため半田付けのテンプラがないかご確認いただき、
下記画像の同色どうしが導通しているかご確認ください。
その他のchocソケットや右手側も同様に上下の端子どうしが導通しているはずですので、ひとつずつご確認いただき、もし導通していないようでしたら、配線が傷ついた可能性がありますので、エナメル線等でつなげていただければ入力できると思います。

お手数ですが、ご確認・ご対応お願いいたします。

![KB39L_chocfix](https://github.com/Yowkees/keyball/assets/84209772/74821647-d8d7-4342-bccb-3a0a3d1846f1)

**Reply by kodomon** [reply]
ご教授いただきありがとうございます。
お恥ずかしい限りですが、改めて確認してみると、「表側からピンセットでショートさせても反応がありません。」は誤りで、導通しておりました。（キーソケットのはんだ付けは問題ありませんでした。）
ただ単に誤ってロープロファイルのキースイッチを使っているのに、トッププレートとボトムプレートの間にミドルアクリルプレートを挟んでしまい、キースイッチの端子がキーソケットに届いていませんでした。
この度はお騒がせして申し訳ありませんでした。ありがとうございました。
